### ISSUE TITLE
Support for AHT25 sensor

### ISSUE THREAD

**Issue by franqenstein** [esphome/feature-requests#1811]
Hello, kindly ask to add support for AHT25 sensor

**Comment by convicte** [comment]
I'd also be interested in having it supported.

Thank you in advance!

**Comment by AlanADV** [comment]
Any news regarding this? the sensor seems quite cheap and high performance.
Thanks!

**Comment by DennisYildirim** [comment]
Oops still no aht25 support :(

**Comment by gtbuchanan** [comment]
The [ESPHome docs](https://esphome.io/components/sensor/aht10.html) mention AHT20/21 so maybe AHT25 is implicitly supported (related PR [here](https://github.com/esphome/feature-requests/issues/1558)). Has anybody tried using the `aht10` platform for the AHT25?

**Comment by jnnsrctr** [comment]
@gtbuchanan Yep, I did, I tried platform `aht10` and variant `AHT20` and it works like a charm.
